[2021-08-08 08:44:39,183] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: presentation_pipeline.presentation_pipeline_start 2021-08-08T08:44:38.735630+00:00 [queued]>
[2021-08-08 08:44:39,197] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: presentation_pipeline.presentation_pipeline_start 2021-08-08T08:44:38.735630+00:00 [queued]>
[2021-08-08 08:44:39,197] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-08 08:44:39,198] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-08 08:44:39,198] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-08 08:44:39,205] {taskinstance.py:1107} INFO - Executing <Task(ExternalTaskSensor): presentation_pipeline_start> on 2021-08-08T08:44:38.735630+00:00
[2021-08-08 08:44:39,213] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'presentation_pipeline', 'presentation_pipeline_start', '2021-08-08T08:44:38.735630+00:00', '--job-id', '58', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/presentation_pipeline.py', '--cfg-path', '/tmp/tmpk_u9nlwt', '--error-file', '/tmp/tmpy7hoimvn']
[2021-08-08 08:44:39,209] {standard_task_runner.py:52} INFO - Started process 1467 to run task
[2021-08-08 08:44:39,214] {standard_task_runner.py:77} INFO - Job 58: Subtask presentation_pipeline_start
[2021-08-08 08:44:39,255] {logging_mixin.py:104} INFO - Running <TaskInstance: presentation_pipeline.presentation_pipeline_start 2021-08-08T08:44:38.735630+00:00 [running]> on host 5e970160fe16
[2021-08-08 08:44:39,297] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=presentation_pipeline
AIRFLOW_CTX_TASK_ID=presentation_pipeline_start
AIRFLOW_CTX_EXECUTION_DATE=2021-08-08T08:44:38.735630+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-08T08:44:38.735630+00:00
[2021-08-08 08:44:39,299] {external_task.py:152} INFO - Poking for standardized_pipeline.standardized_pipeline_end on 2021-08-08T08:44:38.735630+00:00 ... 
[2021-08-08 08:44:49,277] {local_task_job.py:198} WARNING - State of this instance has been externally set to success. Terminating instance.
[2021-08-08 08:44:49,279] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 1467
[2021-08-08 08:44:49,280] {taskinstance.py:1284} ERROR - Received SIGTERM. Terminating subprocesses.
[2021-08-08 08:44:49,307] {process_utils.py:66} INFO - Process psutil.Process(pid=1467, status='terminated', exitcode=0, started='08:44:39') (1467) terminated with exit code 0
